### ISSUE TITLE
crash DT of the fact bad order in the copy of the development.

### ISSUE THREAD

**Issue by cjose38** [darktable-org/darktable#13276]
it is indeed a bug. W10 version 4.2 OM-1
video to reproduce it: https://youtu.be/k8dNT-kVHuE

on the initial development, there is a module "diffusion and sharpness" used with a mask, a second session is used with the raster mask of the first one.
when copying, the instances are inverted => the raster mask is not generated (since it is created afterwards due to the inversion).
there may be a call to the mask that cannot be completed.
the crash is done in cropping, but also in zooming, and the export is crashed on the way:

the cause is identified because when I put the cessions back in the right order, there is no longer the crash.

link with french forum : https://forums.darktable.fr/showthread.php?tid=6349

a raw and xmp are here: https://transfert.free.fr/y7wvjk

Translated with www.DeepL.com/Translator (free version)

**Comment by Nilvus** [comment]
Copy/paste a comment in a french forum will not help. Many developers that could deal with that are not french and will not spent time to read (and so copy/paste previous comment on Deepl) all comments before that comment in french forum. Remember that all developers works freely on their free time. We provide an issue template and you miss nearly all asked informations (a (more) precise step-by-step and your setup configuration, especially graphic card, GPU.

For RAW and XMP, please provide the link here, not ask developer to go on french forum please.

Even if you know that I could answer in french, I do not do that here on purpose. Please help developers to help you and fix that.

**Comment by Nilvus** [comment]
And a clearer title would help too.

**Comment by cjose38** [comment]
We provide an issue template and you miss nearly all asked informations (a (more) precise step-by-step 

I put a video step by step to show the problem.
it is silent.

**Comment by cjose38** [comment]
> If you don’t find it worth to spend a few minutes to document the issue, it seems the issue hasn’t any value to you. So why a developer should spend time to watch a video ?

Hello,
I spent hours, days looking for why the software was crashing. since the phenomenon was not reproduced by users on linux (return from the French forum), I also looked at my configuration.
I opened the xmp files of the photo that didn't crash and the one that did. Without being a computer scientist, I looked for the differences... an explanation.
I tested dozens of things, and each time DT crashed and I had to reopen it...
I have, I think, found the origin of the problem. I opened the post for the anomaly with explanations that seemed sufficient. I took the time to make a video that reproduces the defect rather succinctly because it lasts 1:41
even if the developers are volunteers, I don't understand the lawsuit against me. I thought I was doing the right thing and making progress.
on my side, I'm also a volunteer in my photo club teaching others to use Darktable.
my english is basic, my computer language dates back to fortran in 1980 and I did my best.
I'm not looking for polemics and I won't continue this discussion.
